src/create.cpp

```diff
@@ -562,10 +562,20 @@ namespace create {
     uint8_t cmd[2] = { OC_PLAY, songNumber };
     return serial->send(cmd, 2);
   }
+  
+    bool Create::isLeftWheel() const {
+    if (data->isValidPacketID(ID_BUMP_WHEELDROP)) {
+      return (GET_DATA(ID_BUMP_WHEELDROP) & 0x08) != 0;
+    }
+    else {
+      CERR("[create::Create] ", "Wheeldrop sensor not supported!");
+      return false;
+    }
+  }
 
-  bool Create::isWheeldrop() const {
```

Comment:
True I was thinking on terms of create_autonomy where we want to expose everything to the ROS environment and there is little need for isCliff or isWheeldrop. I shall add those functions back since libcreate can be used independently from ROS.